testsuite/features/support/cobbler_test.rb

```diff
@@ -139,6 +222,13 @@ def exists(what, key, value)
     result
   end
 
+  ##
+  # It takes a string, a string, and a string, and returns a string
+  #
+  # Args:
+  #   what: the type of object you want to get.  This can be one of the following:
```

Comment:
```suggestion
  #   what: the type of object you want to get. This can be one of the following:
```
We should also add a e.g. TODO here so we remember that there is something missing.

Reply:
@srbarrios I saw you edited a line in this, would you happen to maybe know the possible types?

Reply:
```suggestion
  #   what: The type of object you want to get. This can be one of the following:
```